utils/generate_types/overrides.d.ts

```diff
@@ -40,7 +40,6 @@ type PageFunctionOn<On, Arg2, R> = string | ((on: On, arg2: Unboxed<Arg2>) => R
 type SmartHandle<T> = T extends Node ? ElementHandle<T> : JSHandle<T>;
 type ElementHandleForTag<K extends keyof HTMLElementTagNameMap> = ElementHandle<HTMLElementTagNameMap[K]>;
 type HTMLOrSVGElement = SVGElement | HTMLElement;
-type HTMLOrSVGElementHandle = ElementHandle<HTMLOrSVGElement>;
```

Comment:
Let's also remove Serializable and inline HTMLOrSVGElement?

Reply:
Serializable is used by our docs and I'm not ready to remove it now. I think its helpful for documentation, and doesn't hurt anyone.

Inlined HTMLOrSVGElement.